apps/dashboard/app/views/projects/show.html.erb

```diff
@@ -47,6 +48,21 @@
               class: 'btn btn-primary mx-1',
             )
             %>
+
+            <%-
+              params = script.smart_attributes.map { |attr| ["batch_connect_session_context[#{attr.id}]", attr.value] }.to_h
```

Comment:
```suggestion
              params = script.smart_attributes.map { |attr| ["script[#{attr.id}]", attr.value] }.to_h
```

You're sending `batch_connect_session_context` parameters which are being (correctly) filtered by the controllers.

You also likely need to merge with the current master branch because that now has the functionality to save accounts & clusters - 2 fields which are required.  Now that they're serialized in the yaml, you should be able to pull their values here in `script.smart_attributes`.

Reply:
Ok, I'll bring the branch up to date with the changes made since I began so it will work using this setup.